common/lib/xmodule/xmodule/course_module.py

```diff
@@ -321,12 +344,16 @@ class CourseFields(object):
     )
     html_textbooks = List(
         display_name=_("HTML Textbooks"),
-        help=_("For HTML textbooks that appear as separate tabs in the courseware, enter the name of the tab (usually the name of the book) as well as the URLs and titles of all the chapters in the book."),
+        help=_(
+            "For HTML textbooks that appear as separate tabs in the courseware, enter the name of the tab (usually "
+            "the name of the book) as well as the URLs and titles of all the chapters in the book."
+        ),
         scope=Scope.settings
     )
     remote_gradebook = Dict(
         display_name=_("Remote Gradebook"),
-        help=_("Enter the remote gradebook mapping. Only use this setting when REMOTE_GRADEBOOK_URL has been specified."),
+        help=_("Enter the remote gradebook mapping. Only use this setting when "
+               "REMOTE_GRADEBOOK_URL has been specified."),
```

Comment:
Can you make the style of this change match the style of the other changes in this pull request? That is, the string should begin on the next line from the open paren, indented, and the closing paren should be on its own line.